src/tax-lot/tax-lot.repository.ts

```diff
@@ -70,6 +77,100 @@ export class TaxLotRepository {
     }
   }
 
+  async findGeomFromGeoJson(
```

Comment:
Similar to the other commit on `checkGeomIsValid`, could the return type of this just be `Geom`?